.circleci/config.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           command: |
             python -m venv venv
             . venv/bin/activate
-            pip install -e git+https://github.com/plotly/dash.git#egg=dash[testing]
+            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            cd dash-main && pip install -e .[dev,testing] --progress-bar off && cd ..
+            cd dash-main/\@plotly/dash-generator-test-component-nested && npm ci && npm run build && sudo R CMD INSTALL . && cd ../../..
```

Comment:
Using `git clone` and local pip install as we now also need to build the test components